fixtures/martin/rush-blinder-1-ww.json

```
@@ -0,0 +1,140 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "RUSH BLINDER 1 WW",
+  "categories": ["Blinder", "Dimmer", "Strobe"],
+  "meta": {
+    "authors": ["Dark Llama"],
+    "createDate": "2020-06-01",
+    "lastModifyDate": "2020-06-01"
+  },
+  "comment": "Some of the control/shutter channels are not complete due to no matching parameters in OFL.",
+  "links": {
+    "manual": [
+      "https://www.martin.com/en/site_elements/rush-blinder-1-ww-user-manual"
+    ],
+    "productPage": [
+      "https://www.martin.com/en/products/rush-blinder-1-ww"
+    ]
+  },
+  "physical": {
+    "dimensions": [441, 177, 400],
+    "weight": 9.5,
+    "power": 448,
+    "DMXconnector": "3-pin and 5-pin",
+    "bulb": {
+      "type": "2x2 100W COB WW LEDs"
+    },
+    "lens": {
+      "degreesMinMax": [70, 70]
+    }
+  },
```

```suggestion
  },
  "matrix": {
    "pixelKeys": [
      [
        ["1", "2"],
        ["3", "4"]
      ]
    ],
    "pixelGroups": {
      "All": "all"
    }
  },
  "templateChannels": {
    "Dimmer $pixelKey": {
      "capability": {
        "type": "ColorIntensity",
        "color": "Warm White"
      }
    }
  },
```

Is this order correct?  I can't seem to find any documentation on what 1/2/3/4 are.